src/main/scala/sbtassembly/AssemblyPlugin.scala

```diff
@@ -42,90 +44,80 @@ object AssemblyPlugin extends sbt.AutoPlugin {
     packageOptions in assembly := {
       val os = (packageOptions in (Compile, packageBin)).value
       (mainClass in assembly).value map { s =>
-        Package.MainClass(s) +: (os filterNot {_.isInstanceOf[Package.MainClass]})
-      } getOrElse {os}
+        Package.MainClass(s) +: (os filterNot { _.isInstanceOf[Package.MainClass] })
+      } getOrElse os
     },
-    packageOptions in assemblyPackageScala      := (packageOptions in (Compile, packageBin)).value,
+    packageOptions in assemblyPackageScala := (packageOptions in (Compile, packageBin)).value,
     packageOptions in assemblyPackageDependency := (packageOptions in (Compile, packageBin)).value
   )
 
   lazy val baseAssemblySettings: Seq[sbt.Def.Setting[_]] = (Seq(
     assembly := Assembly.assemblyTask(assembly).value,
-    assembledMappings in assembly                   := Assembly.assembledMappingsTask(assembly).value,
-    assemblyPackageScala                            := Assembly.assemblyTask(assemblyPackageScala).value,
-    assembledMappings in assemblyPackageScala       := Assembly.assembledMappingsTask(assemblyPackageScala).value,
-    assemblyPackageDependency                       := Assembly.assemblyTask(assemblyPackageDependency).value,
-    assembledMappings in assemblyPackageDependency  := Assembly.assembledMappingsTask(assemblyPackageDependency).value,
+    assemblyPackageScala := Assembly.assemblyTask(assemblyPackageScala).value,
+    assemblyPackageDependency := Assembly.assemblyTask(assemblyPackageDependency).value,
 
     // test
-    test in assembly := { () },
+    test in assembly := Unit,
```

Comment:
Is this `scala.Unit`, if so isn't that a companion object for `scala.Unit` type, and not an actual term of `Unit` type?

Reply:
You are right. I changed this, silly, because scalafmt removes the surrounding braces, causing a failure. I will find out how scalafmt can skip this. But for now, I will return it to what it was before.

Reply:
Apparently we can use `test in assembly := {}`, and scalafmt leaves it alone. Is this fine?

Reply:
I guess so.